Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -655,10 +655,14 @@ void ViewTransition::setupDynamicStyleSheet(const AtomString& name, const Captur
 
     // group keyframes
     Ref props = MutableStyleProperties::createEmpty();
-    props->setProperty(CSSPropertyWidth, capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyWidth));
-    props->setProperty(CSSPropertyHeight, capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyHeight));
-    props->setProperty(CSSPropertyTransform, capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyTransform));
-    props->setProperty(CSSPropertyBackdropFilter, capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyBackdropFilter));
+    if (auto width = capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyWidth))
+        props->setProperty(CSSPropertyWidth, width.releaseNonNull());
+    if (auto height = capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyHeight))
+        props->setProperty(CSSPropertyHeight, height.releaseNonNull());
+    if (auto transform = capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyTransform))
+        props->setProperty(CSSPropertyTransform, transform.releaseNonNull());
+    if (auto filter = capturedElement.oldProperties->getPropertyCSSValue(CSSPropertyBackdropFilter))
+        props->setProperty(CSSPropertyBackdropFilter, filter.releaseNonNull());
```

Comment:
Maybe this should use:

```c++
Ref props = capturedElement.oldProperties->copyProperties({CSSPropertyWidth,  CSSPropertyHeight, CSSPropertyTransform, CSSPropertyBackdropFilter});
```

instead?

Reply:
Seems roughly equivalent and cleaner, I would switch to that.